src/frontend/components/UploadField/UploadField.tsx

```diff
@@ -29,12 +29,18 @@ export class UploadField extends React.Component<
 
   render() {
     return (
-      <DropzoneStyled
+      <Dropzone
+        accept={'video/*'}
```

Comment:
The drop zone is also used to upload subtitles

Reply:
huhu yes you are right !

Reply:
Done